docs/core/dependency-loading/trusted-platform-assemblies.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Trusted Platform Assemblies - .NET Core
+description: Overview of the concept of trusted platform assemblies used in .NET Core to populate the default AssemblyLoadContext.
+ms.date: 08/09/2019
+author: sdmaclea
+ms.author: stmaclea
+---
+# Trusted Platform Assemblies
```

Comment:
I think we should have a page for `NATIVE_DLL_SEARCH_DIRECTORIES` too, or generalize this page to include it.